src/main/java/picard/sam/PositionBasedDownsampleSam.java

```diff
@@ -55,41 +45,112 @@
-                "the read came from. The downsampling is based on this position. Results with the exact same input will produce the \n" +
-                "same results.\n" +
-                "\n" +
-                "Note 1: This is technology and read-name dependent. If your read-names do not have coordinate information, or if your\n" +
-                "BAM contains reads from multiple technologies (flowcell versions, sequencing machines) this will not work properly. \n" +
-                "This has been designed with Illumina MiSeq/HiSeq in mind.\n" +
-                "Note 2: The downsampling is not random. It is deterministically dependent on the position of the read within its tile.\n" +
-                "Note 3: Downsampling twice with this program is not supported.\n" +
-                "Note 4: You should call MarkDuplicates after downsampling.\n" +
-                "\n" +
-                "Finally, the code has been designed to simulate sequencing less as accurately as possible, not for getting an exact downsample \n" +
-                "fraction. In particular, since the reads may be distributed non-evenly within the lanes/tiles, the resulting downsampling \n" +
-                "percentage will not be accurately determined by the input argument FRACTION.",
+      summary = "<h3>Summary</h3>\n" +
+              "Class to downsample a SAM/BAM file based on the position of the read in a flowcell. As with DownsampleSam, all the " +
+              "reads with the same queryname are either kept or dropped as a unit." +
+              "\n\n " +
+              "<h3>Details</h3>\n" +
+              "The downsampling is _not_ random (and there is no random seed). It is deterministically determined by the position " +
+              "of each read within its \"tile\". Specifically, it draws an ellipse that covers a FRACTION of the total tile's " +
+              "area and of all the edges of the tile. It uses this area to determine whether to keep or drop the record. Since reads " +
+              "with the same name have the same position (mates, secondary and supplemental alignments), the decision will be the " +
+              "same for all of them. The main concern of this downsampling method is that due to \"optical duplicates\" downsampling " +
+              "randomly can create a result that has a different optical duplciate rate, and therefore a different estimated library " +
+              "size (when running mark-duplicates) THe downsampling method done keeps (physically) close reads together, so that (except " +
+              "for the reads near the boundary of the circle) optical duplicates are kept or dropped as a group. " +
+              "By default the program expects the read names to have 5 or 7 fields separated by colons (:) and it takes the last two " +
+              "to indicate the x and y coordinates of the reads within the \"tile\" from whence it was sequenced. See " +
```

Comment:
Why quote "tile" here? Change "from whence" to just "whence".

Actually, it looks like "whence" is used properly elsewhere,
so this is another consequence of copying lots of randomly formatted
text around.